app/grandchallenge/core/templates/challenge.html

```diff
@@ -65,7 +65,7 @@
                             {% if challenge.use_teams %}
                                 {% if "change_challenge" in challenge_perms or user_is_participant %}
                                     <li class="nav-item">
-                                        <a class="nav-link {% if request.resolver_match.view_name == 'teams:list' %}active{% endif %}"
+                                        <a class="nav-link {% if request.resolver_match.view_name == 'teams:list' or request.resolver_match.view_name == 'teams:create' or request.resolver_match.view_name == 'teams:detail' or request.resolver_match.view_name == 'teams:update' or request.resolver_match.view_name == 'teams:delete'%}active{% endif %}"
```

Comment:
You can probably replace all of these individual clauses with `{% if request.resolver_match.app_name == 'teams' %}` (preferred, if it works) or `{% if request.resolver_match.namespace == 'teams' %}`, similar for the evaluation. See https://docs.djangoproject.com/en/3.2/ref/urlresolvers/#django.urls.ResolverMatch

Reply:
wow, yes, that's much better and it works!